README.md

```diff
@@ -39,10 +39,10 @@ as well as adding a Rackspace tab and a Solutions tab, which provides
 Heat templates for commonly deployed applications.
 * `kibana.yml` - Setup Kibana on the Kibana hosts for the logging dashboard.
 * `logstash.yml` - deploys a logstash host. If this play is used, be sure to
-uncomment the related block in user_extra_variables.yml before this play is
-run and then rerun the appropriate plays in openstack-ansible after this
-play to ensure that rsyslog ships logs to logstash. See steps 11 - 13 below
-for more.
+copy the related block in user_rpco_variables_defaults.yml to
+user_rpco_variables_overrides before this play is run and then rerun the
+appropriate plays in openstack-ansible after this play to ensure that rsyslog
+ships logs to logstash. See steps 11 - 13 below for more.
```

Comment:
I have no idea what that was referencing, I don't know exactly what to call it, but I did change it to reference 2-4 and 3-2 instead (which I think is right).